nixos/modules/services/misc/nzbget.nix

```diff
@@ -39,6 +44,13 @@ in
     services.nzbget = {
       enable = mkEnableOption "NZBGet";
 
+      package = mkOption {
```

Comment:
There is only one option for this package, so why add this option?

Reply:
To allow for overrides. I thought of overriding the package to modify the default config file to allow declarative configuration with the current module.

Reply:
This is what the new `settings` option in #132722 effectively achieves.